libraries/lib-theme/AllThemeResources.h

```diff
@@ -108,6 +108,14 @@ from there.  Audacity will look for a file called "Pause.png".
    DEFINE_IMAGE( bmpRedo, wxImage( 26, 24 ), wxT("Redo"));
    DEFINE_IMAGE( bmpRedoDisabled, wxImage( 26, 24 ), wxT("RedoDisabled"));
 
+   SET_THEME_FLAGS(  resFlagPaired  );
+   DEFINE_IMAGE( bmpCut, wxImage( 20, 20 ), wxT("Cut"));
+   DEFINE_IMAGE( bmpCutDisabled, wxImage( 20, 20 ), wxT("CutDisabled"));
+   DEFINE_IMAGE( bmpCopy, wxImage( 20, 20 ), wxT("Copy"));
+   DEFINE_IMAGE( bmpCopyDisabled, wxImage( 20, 20 ), wxT("CopyDisabled"));
+   DEFINE_IMAGE( bmpPaste, wxImage( 20, 20 ), wxT("Paste"));
+   DEFINE_IMAGE( bmpPasteDisabled, wxImage( 20, 20 ), wxT("PasteDisabled"));
```

Comment:
After this is merged, remember to tell @LWinterberg to update the "sprite sheet" in documentation for users who customize the theme.